codegenerator/cli/npm/envio/src/sources/Evm.res

```diff
@@ -0,0 +1,87 @@
+@get external getNumber: Internal.eventBlock => int = "number"
+@get external getTimestamp: Internal.eventBlock => int = "timestamp"
+@get external getId: Internal.eventBlock => string = "hash"
+
+let cleanUpRawEventFieldsInPlace: Js.Json.t => unit = %raw(`fields => {
+    delete fields.hash
+    delete fields.number
+    delete fields.timestamp
+  }`)
+
+let ecosystem: Ecosystem.t = {
+  name: Evm,
+  blockFields: [
+    "number",
+    "timestamp",
+    "hash",
+    "parentHash",
+    "nonce",
+    "sha3Uncles",
+    "logsBloom",
+    "transactionsRoot",
+    "stateRoot",
+    "receiptsRoot",
+    "miner",
+    "difficulty",
+    "totalDifficulty",
+    "extraData",
+    "size",
+    "gasLimit",
+    "gasUsed",
+    "uncles",
+    "baseFeePerGas",
+    "blobGasUsed",
+    "excessBlobGas",
+    "parentBeaconBlockRoot",
+    "withdrawalsRoot",
+    "l1BlockNumber",
+    "sendCount",
+    "sendRoot",
+    "mixHash",
+  ],
+  transactionFields: [
+    "transactionIndex",
+    "hash",
+    "from",
+    "to",
```

Comment:
Did you check some of the newly available fields on hypersync?

Reply:
Not used yet